### PR TITLE
Add Standard_Microsoft in PSSku

### PR DESCRIPTION
**Pull request by hytao**
## Description

Adding Standard_Microsoft in CDN pricing sku

## Checklist

- [x ] I have read the [_Submitting Changes_](https://github.com/Azure/azure-powershell/blob/preview/CONTRIBUTING.md#submitting-changes) section of [`CONTRIBUTING.md`](https://github.com/Azure/azure-powershell/blob/preview/CONTRIBUTING.md)
- [x ] The title of the PR is clear and informative
- [ x] The appropriate [change log has been updated](https://github.com/Azure/azure-powershell/blob/preview/CONTRIBUTING.md#updating-the-change-log)
- [x ] The PR does not introduce [breaking changes](https://github.com/Azure/azure-powershell/blob/preview/documentation/breaking-changes/breaking-changes-definition.md)
- [x ] If applicable, the changes made in the PR have proper test coverage
- [ ] For public API changes to cmdlets:
    - [ ] the changes have gone through a [cmdlet design review](https://github.com/Azure/azure-powershell-cmdlet-review-pr)
    - [ ] the cmdlet markdown files were [generated using the `platyPS` module](https://github.com/Azure/azure-powershell/blob/preview/documentation/development-docs/help-generation.md)

### CODE REVIEW

File: src/ResourceManager/Cdn/Commands.Cdn/Models/Profile/PSSkuName.cs [hunk 1]
```diff
@@ -20,6 +20,7 @@ public enum PSSkuName
         Premium_Verizon,
         Custom_Verizon,
         Standard_Akamai,
-        Standard_ChinaCdn
+        Standard_ChinaCdn,
```
Comment (markcowl) [inline src/ResourceManager/Cdn/Commands.Cdn/Models/Profile/PSSkuName.cs]: @hytao Does this require any change in the api-version, or does it work in the current api-version?  Can yopu update a test to use the new value?

Comment (praries880) [reply src/ResourceManager/Cdn/Commands.Cdn/Models/Profile/PSSkuName.cs]: @hytao in addition to the test, you will need to add a change description under `CurrentRelease` in this file : https://github.com/Azure/azure-powershell/blob/preview/src/ResourceManager/Cdn/Commands.Cdn/ChangeLog.md#current-release

Comment (hytao) [reply src/ResourceManager/Cdn/Commands.Cdn/Models/Profile/PSSkuName.cs]: Hi @praries880 , I have updated the changelog. Currently I'm having trouble running tests in visual studio. Besides the test, what else should I do? We want to finish this PR ASAP.

Thanks!

File: src/ResourceManager/Cdn/Commands.Cdn/ChangeLog.md [hunk 2]
```diff
@@ -20,11 +20,11 @@
 ## Current Release
 
 ## Version 5.0.5
-* Fixed issue with default resource groups not being set.
-* Updated common runtime assemblies
+* Added Standard_Microsoft in Cdn pricing sku.
```
Comment (praries880) [inline src/ResourceManager/Cdn/Commands.Cdn/ChangeLog.md]: move this to under Current Release

And restore the values under the old version numbers as they were

File: src/ResourceManager/Cdn/Commands.Cdn/Models/Profile/PSSkuName.cs (hunk 1 above)
Comment (praries880) [reply src/ResourceManager/Cdn/Commands.Cdn/Models/Profile/PSSkuName.cs]: @hytao  I was able to run the tests on my desktop without any issues.. I ran them in playback mode on a branch which had no changes to the cdn code/tests.

**Pull request closed without merging**